ide/editor.settings.storage/src/org/netbeans/modules/editor/settings/storage/fontscolors/ColoringStorage.java

```diff
@@ -258,25 +264,23 @@ public void delete(MimePath mimePath, String profile, boolean defaults) throws I
         assert profile != null : "The parameter profile must not be null"; //NOI18N
         
         FileObject baseFolder = FileUtil.getConfigFile("Editors"); //NOI18N
-        Map<String, List<Object []>> files = new HashMap<String, List<Object []>>();
+        Map<String, List<Object []>> files = new HashMap<>();
         SettingsType.getLocator(this).scan(baseFolder, mimePath.getPath(), profile, true, defaults, !defaults, false, files);
         
         assert files.size() <= 1 : "Too many results in the scan"; //NOI18N
 
         final List<Object []> profileInfos = files.get(profile);
         if (profileInfos != null) {
-            FileUtil.runAtomicAction(new FileSystem.AtomicAction() {
-                public void run() throws IOException {
-                    for(Object [] info : profileInfos) {
-                        FileObject settingFile = (FileObject) info[1];
-
-                        // Skip files with wrong type of colorings
-                        if (!type.equals(getColoringFileType(settingFile))) {
-                            continue;
-                        }
+            FileUtil.runAtomicAction((FileSystem.AtomicAction) () -> {
+                for(Object [] info : profileInfos) {
+                    FileObject settingFile = (FileObject) info[1];
 
-                        settingFile.delete();
+                    // Skip files with wrong type of colorings
+                    if (!type.equals(getColoringFileType(settingFile))) {
+                        continue;
                     }
+
+                    settingFile.delete();
```

Comment:
Doing a separate commit with just cosmetic changes (lambdafication, diamond operator, override annotations etc.) would encourage quick reviews in the future :-)

Reply:
Going to do that.

Reply:
The cosmetic changes ended up in a separate commit.